app.css

```diff
@@ -3269,60 +3269,96 @@ strong {
   display: block;
   margin: 2rem auto;
   font-weight: 900;
-  padding: 1rem 0; }
+  padding: 1rem 0;
+  text-transform: uppercase; }
 
-header {
-  padding-top: 2rem;
-  padding-bottom: 2rem; }
-  header .navbar {
-    width: 100%;
+header .notice-bar {
+  height: 70px;
+  background: #26A5D4;
+  color: #fff; }
+  header .notice-bar .notice-text {
+    padding-top: 26px;
+    font-size: 1.5rem;
+    font-weight: 300;
+    width: calc(100% - 200px);
+    margin: auto; }
+    header .notice-bar .notice-text a {
+      color: #fff;
```

Comment:
In other places you are using upper cases and full values ex. #33333, #D85039. We should be consistent.

Reply:
:+1: